### ISSUE TITLE
Missing stream sources after rebooting aircraft

### ISSUE THREAD

**Issue by eric-dk** [dji-sdk/Mobile-SDK-Android-V5#16]
Steps-to-reproduce:
1. Launch `sample-app-aircraft`
2. Turn on Matrice 30 or 30T
3. Tap *Testing Tools*
4. Tap *Multi-Video Decoding*
5. Reboot aircraft while leaving app open

Expected:
Video feed(s) reconnect and resume.

Actual:
On first connection, the following returns at least one `StreamSource`.
```kotlin
MediaDataCenter.getInstance().videoStreamManager.availableStreamSources
```
After rebooting the aircraft, this list is always empty.

**Comment by DJI-William** [comment]
Reproducible, I will raise an internal ticket for the dev team.

**Comment by DJI-William** [comment]
This issue is fixed, wait until MSDK 5.1 version at the end of this month.
